jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterFactory.java

```diff
@@ -95,7 +97,7 @@ public InfluxDbWriterFactory(
 		this.retentionPolicy = StringUtils.isNotBlank(retentionPolicy) ? retentionPolicy : DEFAULT_RETENTION_POLICY;
 
 		this.resultAttributesToWriteAsTags = initResultAttributesToWriteAsTags(resultTags);
-
+		this.tags = tags;
```

Comment:
In case the `tags` param is `null`, `this.tags` should be initialized to an empty map.